lua/xcodebuild/core/xcode.lua

```diff
@@ -512,18 +512,28 @@ end
 ---Installs the application on simulator.
 ---@param destination string
 ---@param appPath string
+---@param retrying boolean|nil
 ---@param callback function|nil
 ---@return number # job id
-function M.install_app_on_simulator(destination, appPath, callback)
+function M.install_app_on_simulator(destination, appPath, retrying, callback)
   local command = "xcrun simctl install '" .. destination .. "' '" .. appPath .. "'"
 
   return vim.fn.jobstart(command, {
     stdout_buffered = true,
     on_exit = function(_, code, _)
       if code ~= 0 then
-        notifications.send_error("Could not install app (code: " .. code .. ")")
         if code == 149 then
```

Comment:
Please restore this notification when the code is different than 149:
```
notifications.send_error("Could not install app (code: " .. code .. ")")
```

Because otherwise there is no notification to the user if something goes wrong.
I think this would be the last change and we are ready to go.

Reply:
@josejuanqm would you like to fix it :)? If not I will take over this PR to close it

Reply:
Apologies! I'll fix this tonight